<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ScoreWindow</class>
 <widget class="QMainWindow" name="ScoreWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>786</width>
    <height>523</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ScoreWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QWidget" name="output_doc_widget" native="true"/>
      <widget class="QWidget" name="preview_widget" native="true">
       <layout class="QVBoxLayout" name="verticalLayout_2" stretch="0,1">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QToolButton" name="toolButton_2">
            <property name="text">
             <string>-</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="toolButton_4">
            <property name="text">
             <string>+</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="toolButton_3">
            <property name="text">
             <string>CW</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="toolButton">
            <property name="text">
             <string>CCW</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="toolButton_5">
            <property name="text">
             <string>&lt;</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="toolButton_6">
            <property name="text">
             <string>&gt;</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="PdfView" name="preview_pdfview" native="true"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="files_widget" native="true">
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QLabel" name="path_label">
          <property name="text">
           <string>/path/to/files</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="filter_lineEdit">
          <property name="placeholderText">
           <string>.*pdf</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QListWidget" name="files_listWidget"/>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>786</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="save_action"/>
   <addaction name="open_action"/>
   <addaction name="export_action"/>
  </widget>
  <action name="save_action">
   <property name="icon">
    <iconset theme="document-save"/>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
   <property name="toolTip">
    <string>Save file list</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="open_action">
   <property name="icon">
    <iconset theme="folder-open"/>
   </property>
   <property name="text">
    <string>Open</string>
   </property>
   <property name="toolTip">
    <string>Open folder</string>
   </property>
  </action>
  <action name="export_action">
   <property name="icon">
    <iconset theme="document-print"/>
   </property>
   <property name="text">
    <string>Export</string>
   </property>
   <property name="toolTip">
    <string>Export as PDF</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PdfView</class>
   <extends>QWidget</extends>
   <header>pdfview.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
